docs/source/conf.py

```diff
@@ -340,3 +341,41 @@
 
 spelling_lang = 'en_US'
 spelling_word_list_filename = 'spelling_wordlist.txt'
+
+
+def _import_object_from_name(module_name, fullname):
+    obj = sys.modules.get(module_name)
+    if obj is None:
+        return None
+    for comp in fullname.split('.'):
+        obj = getattr(obj, comp)
+    return obj
+
+
+def linkcode_resolve(domain, info):
+    if domain != 'py' or not info['module']:
+        return None
+
+    tag = 'v{}'.format(__version__)
```

Comment:
Yes, the branch policy has changed since I wrote that.
Could you elaborate your concern?
Say we published the documentation of v2.0.1. In my understanding, it will always point to the fixed source code on GitHub tagged as v2.0.1, regardless of the status of the development branch. (I assume "latest" is the same meaning as "v2.0.1" at this moment)